README.md

```diff
@@ -411,65 +440,585 @@ Four actions are available for you to hook onto through the tooltip/popover life
+- [assertTooltipVisible()](#asserttooltipvisible)
+- [assertTooltipNotVisible()](#asserttooltipnotvisible)
+- [assertTooltipSide()](#asserttooltipside)
+- [assertTooltipSpacing()](#asserttooltipspacing)
+- [triggerTooltipTargetEvent()](#triggertooltiptargetevent)
+
+All assert helpers require `assert` to be passed as the first param and accept a second, optional param for additional test options. For detailed usage instructions and examples, see the documentation for each test helper below.
+
+All test helpers can be imported from the following path:
+
+```js
+'appname/tests/helpers/ember-tooltips';
+```
+
+For example:
+
+```js
+// appname/tests/integration/components/some-component.js
+
+import {
+  assertTooltipRendered,
+} from 'appname/tests/helpers/ember-tooltips';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent('some-component', 'Integration | Component | Some', {
+  integration: true,
+});
+
+test('tooltip-on-element animates with a delay', function(assert) {
+
+  // ... Test content...
+
+  assertTooltipRendered(assert);
+
+  // ... More test content...
+
+});
+```
+
+#### assertTooltipContent()
+
+Asserts that a tooltip or popover has content that matches a given string.
+
+```js
+import {
+  assertTooltipContent,
+} from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`{{tooltip-on-element text='More info'}}`);
+
+  assertTooltipContent(assert, {
+    contentString: 'More info',
+  });
+});
+```
+
+The [options hash](#test-helper-options) accepts:
+
+- [`contentString`](#test-helper-option-contentstring) - REQUIRED
+- [`selector`](#test-helper-option-selector)
+
+#### assertTooltipRendered()
+
+Asserts that a tooltip or popover has been rendered in the DOM.
+
+```js
+import {
+  assertTooltipRendered,
+} from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`{{tooltip-on-element}}`);
+
+  assertTooltipRendered(assert);
+});
+```
+
+This does not assert that the tooltip or popover is visible to the user - use [assertTooltipVisible()](#assertTooltipVisible) for that.
+
+Given this addon's lazy rendering capabilities (explained in [Targets](#targets)), tooltips may not be rendered until the target is interacted with. As such, this helper is often used in conjunction with [triggerTooltipTargetEvent()](#triggerTooltipTargetEvent) to test those user events.
+
+For example:
+
+```js
+import {
+  assertTooltipRendered,
+  triggerTooltipTargetEvent,
+} from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`
+    {{tooltip-on-element enableLazyRendering=true}}
+  `);
+
+  /* Tooltip won't be rendered in the DOM yet because enableLazyRendering delays the rendering until the user interacts with the target */
+
+  triggerTooltipTargetEvent($(this), 'mouseenter');
+
+  /* Now the user has interacted with the target, so the tooltip should be visible... */
```

Comment:
`/* Now the user has interacted with the target, so the tooltip should be RENDERED... */`